package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "SalesforceMobileSDK-CordovaPlugin",
+    "version": "5.2.0",
+    "description": "Salesforce Mobile SDK Cordova Plugin Repository",
+    "keywords": [ "mobilesdk", "android", "ios", "cordova", "salesforce", "mobile", "sdk" ],
+    "homepage": "https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin",
+    "bugs": "https://github.com/forcedotcom/SalesforceMobileSDK-CordovaPlugin/issues",
+    "licenses" : [
+        { "type": "Salesforce.com Mobile SDK License", "url": "https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/master/LICENSE.md" }
```

Comment:
Should we have the license file in the plugin repo as well (like we do for the ios and android repos)?